VAMobile/src/components/CategoryLandingAlert.tsx

```diff
@@ -1,24 +1,38 @@
 import React, { FC } from 'react'
 import { useTranslation } from 'react-i18next'
+import { HapticFeedbackTypes } from 'react-native-haptic-feedback'
 
 import { Box, TextView, VAIcon } from 'components'
 import { NAMESPACE } from 'constants/namespaces'
+import { triggerHaptic } from 'utils/haptics'
 import { useTheme } from 'utils/hooks'
 
 interface CategoryLandingAlertProps {
   /** Text for alert */
   text: string
   /** Optional boolean for indicating an error by setting the text color to red */
   isError?: boolean
+  /** Boolean to determine whether haptics should fire */
+  bypassHaptics: boolean
```

Comment:
I'd drop the bypass at this point since we have no reason to use it currently.

Reply:
fixed!